### ISSUE TITLE
Error loading example config

### ISSUE THREAD

**Issue by zhengtianbao** [prometheus-community/json_exporter#179]
When I follow the readme guide to run the example, json_exporter report error message:
```
caller=main.go:59 level=error msg="Error loading config" err="yaml: line 11: did not find expected key"
```
https://github.com/prometheus-community/json_exporter/blob/9ecc44b0645c8f8e786244a3b4bb96a24c86eb29/examples/config.yml#L12

Replace the outermost double quotes with single quotes at Line 12, then it worked.

**Comment by turtletree** [comment]
Had the same issue and the above suggested solution fixed it. Thanks :) @zhengtianbao 
Please include the fix into the main branch, thanks!

**Comment by rustycl0ck** [comment]
Fixed in #185
